Formula/pypy3.rb

```diff
@@ -1,10 +1,17 @@
 class Pypy3 < Formula
   desc "Implementation of Python 3 in Python"
   homepage "https://pypy.org/"
-  url "https://downloads.python.org/pypy/pypy3.7-v7.3.9-src.tar.bz2"
-  sha256 "70426163b194ee46009986eea6d9426098a3ffb552d9cdbd3dfaa64a47373f49"
   license "MIT"
-  head "https://foss.heptapod.net/pypy/pypy", using: :hg, branch: "py3.7"
+  head "https://foss.heptapod.net/pypy/pypy", using: :hg, branch: "py3.8"
+
+  stable do
+    # M1 support is planned to be included in the next release
+    # See https://www.pypy.org/posts/2022/07/m1-support-for-pypy.html
+    depends_on arch: :x86_64
```

Comment:
Will they release a version of `pypy` with M1 support as well? Or is there a way of bootstrapping this without `pypy`?

Reply:
PyPy seems to share the base code for all Python versions (3.x and 2.7), so there will likely be the M1 build for `pypy` on the next release.